build.sbt

```diff
@@ -21,4 +21,13 @@ sys.env.getOrElse("JAVA_OPTS", "").split(" ").toSeq.map { opt =>
   javaOptions in reStart += opt
 }
 
+inThisBuild(
```

Comment:
Used to run the scalafix. I can remove if we don't want it still in there.

Reply:
I think we should remove it once its usefulness has expired. Is it useful beyond the cats migration?